scipy/stats/__init__.py

```diff
@@ -300,6 +300,7 @@
    jarque_bera
    page_trend_test
    tukey_hsd
+   poisson_means_test
```

Comment:
I ran across [this](https://stackoverflow.com/questions/33944914/implementation-of-e-test-for-poisson-in-python) which says that the test is now in`statsmodels` : / Is it the same?

If so, I hate to say it, but we should probably email the mailing list about this before going too much further : ( Are there advantages of this implementation?

We _can_ have the same test in both (e.g. Tukey's HSD) if it is fundamental enough to belong in SciPy. @josef-pkt what do you think?